scalingo/resource_scalingo_scm_integration.go

```diff
@@ -0,0 +1,126 @@
+package scalingo
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	scalingo "github.com/Scalingo/go-scalingo/v5"
+)
+
+func resourceScalingoScmIntegration() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceScmIntegrationCreate,
+		ReadContext:   resourceScmIntegrationRead,
+		DeleteContext: resourceScmIntegrationDelete,
+
+		Schema: map[string]*schema.Schema{
+			"scm_type": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"url": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"access_token": {
+				Type:      schema.TypeString,
+				Required:  true,
+				ForceNew:  true,
+				Sensitive: true,
+			},
+			"uid": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"username": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"avatar_url": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"email": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"profile_url": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceScmIntegrationImport,
+		},
+	}
+}
+
+func resourceScmIntegrationCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	client, _ := meta.(*scalingo.Client)
+
+	scmType, _ := d.Get("scm_type").(string)
+	url, _ := d.Get("url").(string)
+	accessToken, _ := d.Get("access_token").(string)
```

Comment:
Nipick: Is there a reason to use `_` as it is not mandatory?

By the way I'd add the `ok` syntax to be sure that the cast is working, WDYT?

Reply:
I think it's a good idea ! As this issue is present in the whole repository, do you mind if I fix this in another PR ?

Reply:
As the value types are checked by the schema itself, this may not be so relevant

Reply:
I agree, so looks good.